libsolidity/formal/CHC.cpp

```diff
@@ -990,13 +990,13 @@ vector<smtutil::Expression> CHC::initialStateVariables(ContractDefinition const&
 	return stateVariablesAtIndex(0, _contract);
 }
 
-vector<smtutil::Expression> CHC::stateVariablesAtIndex(int _index)
+vector<smtutil::Expression> CHC::stateVariablesAtIndex(unsigned _index)
```

Comment:
@leonardoalt can you check this? `stateVariablesAtIndex` uses `unsigned`.

Reply:
@axic `unsigned` should be fine